<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tixo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STEP</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="memView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>0x0000: 00000 00000 00000 00000 00000 00000 00000 00000
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="regView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pokeAddr">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ADDRESS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pokeData">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>DATA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poke">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POKE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="vector">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="Line" name="decoline">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="flexo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>409</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GWMonospace</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="tape">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>121</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GWMonospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'GWMonospace'; font-size:11pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tapeLoad">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>480</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOAD TAPE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tapeSave">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>510</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAVE TAPE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memSave">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAVE CORE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memLoad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOAD CORE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="peek">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caliban</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PEEK</string>
    </property>
   </widget>
   <zorder>restart</zorder>
   <zorder>step</zorder>
   <zorder>memView</zorder>
   <zorder>regView</zorder>
   <zorder>pokeData</zorder>
   <zorder>poke</zorder>
   <zorder>vector</zorder>
   <zorder>startStop</zorder>
   <zorder>decoline</zorder>
   <zorder>flexo</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>tape</zorder>
   <zorder>tapeLoad</zorder>
   <zorder>tapeSave</zorder>
   <zorder>memSave</zorder>
   <zorder>memLoad</zorder>
   <zorder>mode</zorder>
   <zorder>pokeAddr</zorder>
   <zorder>peek</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
